### ISSUE TITLE
ESLint error: AssertionError [ERR_ASSERTION]: 'baseToken' should not be null or undefined.

### ISSUE THREAD

**Issue by lwswnnscrm** [vuejs/eslint-plugin-vue#1261]
**Tell us about your environment**
* **ESLint version:** 5.15.1
* **eslint-plugin-vue version:** 5.2.2
* **Node version:** 14.5.0/14.4.0

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  extends: [
    'plugin:vue/recommended',
    'airbnb-base',
  ],
  plugins: [
    'vue',
    'html',
  ],
  rules: {
    'import/no-unresolved': 'off',
    'no-undef': 'off',
    'no-shadow': 'off',
    'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['state', 'app'] }],
    'import/no-extraneous-dependencies': 'off',
    'no-underscore-dangle': 'off',
    'vue/no-v-html': 'off',
    'max-len': 'off',
  },
}

```

**What did you do?**	
When i start command ```eslint --ext .js,.vue --fix --ignore-path .gitignore .``` or ```eslint \"./**/*.{vue, js}\" --fix --ignore-pattern ./node_modules``` i have error its happen only if i use windows(in my pc windows 10), mac os all ok.

**What actually happened?**
Error that i see ```AssertionError [ERR_ASSERTION]: 'baseToken' should not be null or undefined.
    at setOffset (C:\ssfront-nuxt\node_modules\eslint-plugin-vue\lib\utils\indent-common.js:292:5)
    at processNodeList (C:\ssfront-nuxt\node_modules\eslint-plugin-vue\lib\utils\indent-common.js:468:7)
    at EventEmitter.CallExpression (C:\ssfront-nuxt\node_modules\eslint-plugin-vue\lib\utils\indent-common.js:1060:7)
    at EventEmitter.emit (events.js:314:20)
    at NodeEventGenerator.applySelector (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:3128:26)
    at NodeEventGenerator.applySelectors (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:3142:22)
    at NodeEventGenerator.enterNode (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:3150:14)
    at traverse (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:110:13)
    at traverse (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:122:13)
    at traverse (C:\ssfront-nuxt\node_modules\vue-eslint-parser\index.js:117:21)```

**Comment by ota-meshi** [comment]
Please share the actual source code causing the issue.

**Comment by raul338** [comment]
I'm having the same problem with a large project, on linux runs fine (build and serve). On windows it fails on both with this error. I'll try to reproduce a small example.

**Comment by ota-meshi** [comment]
Please open the issue with the content requested in the issue template.
